inc/namespace.php

```diff
@@ -425,7 +425,12 @@ function ( $error ) {
 		'stats'     => $stats,
 	];
 
-	$trace['metadata'] = redact_metadata( $metadata );
+	$annotations = [
+		'memoryUsage' => round( memory_get_peak_usage() / 1048576, 2 ), // Convert bytes to mb.
```

Comment:
Can we send the actual value and process it on output instead?

Reply:
Counter point is that being able to include this in the filter expressions, having it atleast as a mb value would be beneficial. Maybe not the rounding though

Reply:
I considered this but I think that caused an issue in UX when you actually try searching in X-ray using annotations, You will see values in MB in the UI but when searching you will have to convert it to KB.

Let me know what you think

Reply:
I'd say we can convert the units in the filtering side if we want? e.g. `annotations.memoryUsage > 2M` or `2G` etc

Reply:
I think it's better to stay native, else we'd need to transform the filter expression further. Also I don't think we'll ever need anything other than MB units for memory usage

Reply:
Should I remove rounding from the conversion?

Reply:
Removed the rounding-off of the used memory to get the exact value of memory consumption